pkg/term/windows/windows.go

```diff
@@ -3,3 +3,27 @@
 // and return pseudo-streams that convert ANSI sequences to / from Windows Console API calls.
 
 package windows
+
+import (
+	"io/ioutil"
+	"os"
+
+	ansiterm "github.com/Azure/go-ansiterm"
+	"github.com/Sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+func init() {
+	logFile := ioutil.Discard
+
+	if isDebugEnv := os.Getenv(ansiterm.LogEnv); isDebugEnv == "1" {
+		logFile, _ = os.Create("ansiReaderWriter.log")
```

Comment:
I'd rather not do this on init.
Just by virtue of importing this package this gets created.

Can we throw this in a normal function and call it (and populate as needed) from both functions?